packages/jest-types/__typetests__/jest.test.ts

```diff
@@ -413,6 +413,8 @@ expectError(jest.clearAllTimers(false));
 expectType<number>(jest.getTimerCount());
 expectError(jest.getTimerCount(true));
 
+expectType<number>(jest.now());
```

Comment:
Perhaps let’s add a check that `jest.now()` does not take args? Not the best idea to test this indirectly, `tsd` does not have a way to test type of args at this moment. 

```suggestion
expectType<number>(jest.now());
expectError(jest.now('1995-12-17T03:24:00'));
```